app/exec/extension/default.ts

```diff
@@ -30,6 +30,8 @@ export interface ExtensionArguments extends CoreArguments {
 	extensionName: args.StringArgument;
 	json5: args.BooleanArgument;
 	locRoot: args.ExistingDirectoriesArgument;
+	manifestJs: args.StringArgument;
```

Comment:
should be `args.ReadableFilePathsArgument`

Reply:
Done.